data-service/src/main/java/uk/gov/gchq/palisade/service/data/web/DataController.java

```diff
@@ -80,7 +80,7 @@ public DataController(final AuditableDataService auditableDataService, final Aud
         AuditErrorMessage authorisationErrorMessage = auditableAuthorisedDataRequest.getAuditErrorMessage();
 
         if (authorisationErrorMessage != null) {
-            LOGGER.error("Error occurred processing the authoriseRequest for  {}", authorisationErrorMessage);
+            LOGGER.error("Error occurred processing the authoriseRequest error message was {}", authorisationErrorMessage);
```

Comment:
```suggestion
            LOGGER.error("Error occurred processing the authoriseRequest, error message was: {}", authorisationErrorMessage);
```
Maybe the "error message was" could be removed?

Reply:
Looked into it and a few things that could change.  First on the list is that it should have the Exception not the AuditErrorMessage.  Re-worded the message to be clearer.